src/main/webapp/resources/js/components/sequence-files/SingleEndFileRenderer.jsx

```diff
@@ -60,23 +67,28 @@ export function SingleEndFileRenderer({
                           )
                     }
                     target="_blank"
+                    className="t-file-label"
                   >
                     {file.fileInfo.label}
                   </a>
                 ) : (
-                  <span>{file.fileInfo.label}</span>
+                  <span className="t-file-label">{file.fileInfo.label}</span>
                 )}
 
                 <Space direction="horizontal" size="small">
                   {file.fileType === "assembly"
                     ? null
                     : getProcessingState(file.fileInfo.processingState)}
-                  <span style={{ marginRight: SPACE_XS }}>
+                  <span
+                    style={{ marginRight: SPACE_XS }}
+                    className="t-file-size"
+                  >
                     {file.firstFileSize}
                   </span>
                   <Button
                     shape="circle"
                     icon={<IconDownloadFile />}
+                    className="t-download-file-btn"
                     onClick={() =>
                       file.fileType === "assembly"
```

Comment:
Abstract out into a function or even let a utility file handle it to clean up this component

Reply:
Doesn't really require another file to handle it so I will just put it into a function in the component

Reply:
Moved into a function in a0030449ca44d14e86faa4e3ece5b118bf115cc1